tests/common/plugins/sanity_check/checks.py

```diff
@@ -18,61 +18,17 @@
 cache = FactsCache()
 
 CHECK_ITEMS = [
-    'check_services',
+    'check_processes',
```

Comment:
The `check_service` has a retry logic if `networking_uptime < SYSTEM_STABILIZE_MAX_TIME`. Do we keep this logic after this change?

Reply:
The `check_process` check has the same retry logic: https://github.com/Azure/sonic-mgmt/blob/master/tests/common/plugins/sanity_check/checks.py#L711